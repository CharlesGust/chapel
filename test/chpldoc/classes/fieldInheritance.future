bug: output field type correctly