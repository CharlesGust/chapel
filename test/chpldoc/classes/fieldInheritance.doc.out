Module: fieldInheritance
   Class: Elder
       This class has two fields 
      field rod: string
      field staff: bool
      inherited from object

   Class: Younger
       This class should inherit its Elder's fields 
      field key: int(64)
      field stone: imag(64)
      inherited from Elder
         field rod: string
         field staff: bool

      inherited from object

